core/domain/collection_domain_test.py

```diff
@@ -573,28 +574,71 @@ def test_yaml_import_and_export(self):
             collection_domain.Collection.from_yaml('collection3', None)
 
 
+class SchemaMigrationMethodsUnitTests(test_utils.GenericTestBase):
+    """Tests the presence of appropriate schema migration methods in the
+    Collection domain object class.
+    """
+
+    def test_correct_collection_content_schema_conversion_methods_exist(self):
```

Comment:
What tests are you referring to here? I believe I already have a test which tests that nodes gets migrated to collection_contents.